aws/resource_aws_db_instance.go

```diff
@@ -911,7 +894,14 @@ func resourceAwsDbInstanceCreate(d *schema.ResourceData, meta interface{}) error
 		}
 
 		if attr, ok := d.GetOk("name"); ok {
-			opts.DBName = aws.String(attr.(string))
+			// "Note: This parameter [DBName] doesn't apply to the MySQL, PostgreSQL, or MariaDB engines."
```

Comment:
This section is going to create problems. On _restore_, the `DBName` doesn't apply but will be ignored. However, on _create_, MySQL, Postgres, and MariaDB need to the `DBName`.
https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html